mandatory/3-function-output.js

```diff
@@ -3,14 +3,13 @@ function getRandomNumber() {
   return Math.random() * 10;
 }
 
-// Add comments to explain what this function does. You're meant to use Google!
+// join two strings
 function combine2Words(word1, word2) {
   return word1.concat(word2);
 }
 
 function concatenate(firstWord, secondWord, thirdWord) {
-  // Write the body of this function to concatenate three words together.
-  // Look at the test case below to understand what this function is expected to return.
+ 
 }
```

Comment:
You can add multiple input arguments in concat and make sure to include balnk space beteen words so it can pass the test, like:  firstWord.concat(" ", secondWord, ... )
Well done and nice job!